index.html

```diff
@@ -190,13 +190,13 @@ <h4>JavaScript Open Source Awards</h4>
                         <p><img src="./pic/tile-1-white.svg" width="65" alt="" /></p>
                         <h3>JavaScript Open Source Awards</h3>
                         <p>
-                          JSNation is a JavaScript and Node.js oriented conference, gathering core library authors,
+                          JSNation is a JavaScript oriented conference, gathering core library authors,
                           inspiring engineers and code artists to share their experiences and the vision of the modern
                           development.<br />
-                          In 2021, JSNation will host its third annual JavaScript Open Source Awards ceremony. In the
-                          meantime, take a look at the list of the 2020 winners.
+                          In 2022, JSNation will host its fourth annual JavaScript Open Source Awards ceremony. In the
+                          meantime, take a look at the list of the 2022 winners.
```

Comment:
Here we should mention `take a look at the list of 2021 winners`

Reply:
Thank you for noticing. As it was a typo I just added a new commit to the PR.